app/HutchAgent/Config/LicenseOptions.cs

```diff
@@ -0,0 +1,10 @@
+namespace HutchAgent.Config;
+
+public class LicenseOptions
+{
+  public string Uri { get; set; } = "https://spdx.org/licenses/CC-BY-4.0";
```

Comment:
actually default the license to nothing.

e.g. `string.Empty` for the id/url. `null` for the CreativeWork?